utils/api.ts

```diff
@@ -0,0 +1,54 @@
+import posts from '../config/posts.json';
+import { IPost, Posts } from './types';
+
+/**
+ * Retrieves all posts.
+ * @returns {Object} All posts.
+ */
+export function getAllPosts(): object {
```

Comment:
Instead of providing return type for this as object, can't we make it's type in the format of `Post[]`?

Reply:
No because we are basically returning `posts.json` here, and it is an object